### ISSUE TITLE
False Positive Error for Error: Failed to load plugin 'plugin' declared in '.eslintrc': Cannot find module 'plugin'

### ISSUE THREAD

**Issue by KyleMcKell** [mantoni/eslint_d.js#219]
Currently, I'm running into an issue which is purely unique to `eslint_d` and doesn't persist with `eslint`

The package in question is `eslint-plugin-firebase-functions`.

Easiest way to describe the issue is with a how to reproduce, 

## How to reproduce

Create a new application (I used `npx create-react-app` with default configuration), and create an `.eslintrc`

Paste the following:
```js
// .eslintrc
{
  "plugins": ["firebase-functions"], // or "eslint-plugin-firebase-functions"
}
```
Run commands:
```zsh
~ eslint --version
v8.24.0
~ eslint_d --version
v8.24.0 (eslint_d v12.2.1)
~ eslint .
~ eslint_d .
```
Both as expected throw an error as follows:

### Eslint
```zsh
~ eslint .
Oops! Something went wrong! :(

ESLint: 8.24.0

ESLint couldn't find the plugin "eslint-plugin-firebase-functions".

(The package "eslint-plugin-firebase-functions" was not found when loaded as a Node module from the directory "home/kyle/my-app".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install eslint-plugin-firebase-functions@latest --save-dev

The plugin "eslint-plugin-firebase-functions" was referenced from the config file in ".eslintrc".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.
```
### Eslint_d
```zsh
~ eslint_d .
Error: Failed to load plugin 'firebase-functions' declared in '.eslintrc': Cannot find module 'eslint-plugin-firebase-functions'
Require stack:
- home/kyle/my-app/__placeholder__.js
```
Install the package
```zsh
~ npm install eslint-plugin-firebase-functions@latest --save-dev
```
Run commands again
### Eslint 
Shows a different error since original one is solved
```zsh
~ eslint .
home/kyle/my-app/src/App.js
  1:1  error  Parsing error: The keyword 'import' is reserved

home/kyle/my-app/src/App.test.js
  1:1  error  Parsing error: The keyword 'import' is reserved

home/kyle/my-app/src/index.js
  1:1  error  Parsing error: The keyword 'import' is reserved

home/kyle/my-app/src/reportWebVitals.js
  1:1  error  Parsing error: The keyword 'const' is reserved

/home/kyle/my-app/src/setupTests.js
  5:1  error  Parsing error: The keyword 'import' is reserved
```
### Eslint_d
```zsh
~ eslint_d .
Error: Failed to load plugin 'firebase-functions' declared in '.eslintrc': Cannot find module 'eslint-plugin-firebase-functions'
Require stack:
- /home/kyle/my-app/__placeholder__.js
```

## Operating System
```shell
C:\Users\Kyle>ver

Microsoft Windows [Version 10.0.19043.2006]

C:\Users\Kyle>wsl -l -v
  NAME                   STATE           VERSION
* Ubuntu                 Running         2
  docker-desktop-data    Running         2
  docker-desktop         Running         2
```

##  Node Versioning
```zsh
~ node -v
v18.7.0
~ npm -v 
8.15.0

# also tried on lts
~ node -v
v16.17.1
~ npm -v
8.15.0
```

**Comment by mantoni** [comment]
If you're changing plugins, the eslint_d daemon sometimes needs a manual restart to pick up the change: `eslint_d restart`

Please let me know if the issue still persists after restarting.

**Comment by KyleMcKell** [comment]
Sick, that does seem to be the issue. Sorry for missing that. Closing the issue.
